static_files/index.html

```diff
@@ -29,7 +29,7 @@
           <ul>
             <li><a href="#">Catho Components</a></li>
             <li><a href="#">Quantum</a></li>
-            <li>&#x2630;</li>
+            <li class = "sandwich-button"><a href="#">&#x2630;</a></li>
```

Comment:
In that case, how about replacing the "a" tag with a "button"? I think it makes more sense if we follow the purpose of this item, which is to open a menu and not lead to a new page.

Reply:
Changed to butten and moved the class com li to the button tag